srcpkgs/ansible-base/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'ansible-base'
+pkgname=ansible-base
+version=2.10.2
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="${hostmakedepends} python3-cryptography python3-Jinja2 python3-paramiko
+ python3-yaml"
+checkdepends="python3-pytest"
+short_desc="Simple deployment, configuration management and execution framework"
```

Comment:
Not sure, does it make sense to add a different description here, and if so, which one?